soc/arm/st_stm32/stm32l4/Kconfig.defconfig.stm32l4r5xx

```diff
@@ -11,4 +11,8 @@ config SOC
 config NUM_IRQS
 	default 95
 
+config DMAMUX_STM32
+	default y
+	depends on DMA_STM32
+
```

Comment:
This will conflict with the #30784
As the `drivers/dma/Kconfig.stm32` is configuring the DMAMUX_STM32 for the series